### ISSUE TITLE
http live streaming (MPEG-2 transport stream) output

### ISSUE THREAD

**Issue by bitjeep** [jgh-/VideoCore#180]
Hi.  I was wondering if any work has been done to create an output module + segmenter that would support HTTP live streaming?

**Comment by jgh-** [comment]
No, but if it's something you're looking at it would be a great contribution.  Basically you would want to create a Transform module similar to the RTMP packetizers likely to do the segmenting/muxing to TS format and then perhaps an appropriate output session for whatever you're going to do with that data (be it save to disk or send over a network or whatever).

**Comment by bitjeep** [comment]
Yes, it's something I'm definitely interested in.  I just wanted to ensure that I wasn't duplicating effort before I did too much work on it.  I'm just starting to familiarize myself with the codebase, so thanks for the info about what areas I should be looking at for the structure.

**Comment by tonyisid** [comment]
kickflip use hls, but big problem is delay

**Comment by larrytin** [comment]
@bitjeep @tonyisid Is there any process?
